packages/web/src/pages/app/community.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  Divider,
+  Heading,
+  HStack,
+  Link,
+  Spacer,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { NextPage } from 'next';
+import React from 'react';
+import { FaDiscord } from 'react-icons/fa';
+import { AppLayout } from '../../components/Layout';
+
+// To Do:
+// - move styling to /theme/index
+// - move discord link to env variables
```

Comment:
You won't be able to access env variables from the front-end, so go ahead and hard-code a link for now. We'll add a config endpoint in the future.

Reply:
cool